doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='githooks-prepare-proc-{context}']
+= Preparing Git hooks 
+
+In an authoring environment you can use Git hooks to execute custom operations when the source code of a project in {CENTRAL} is changed. The typical use of Git hooks is interaction with an upstream repository. 
+
+To enable Git hooks to interact with an upstream repository using SSH authentication, you must also provide a secret key and a known hosts file for authentication with the repository.
+
+Skip this procedure if you do not want to configure Git hooks.
+
+.Procedure
+
+. Create the Git hooks files. For instructions, see the https://git-scm.com/docs/githooks[Git hooks reference documentation].
```

Comment:
For minimalism, I suggest: Create the Git hooks files. See https://git-scm.com/docs/githooks[Git hooks reference documentation].

Reply:
In other places we tend to write "for instructions" or "for more information", and for all I know a mere "see..." without clarification is discouraged? I don't think there are written guidelines though